astroquery/alma/core.py

```diff
@@ -678,19 +678,33 @@ def query_sia(self, *, pos=None, band=None, time=None, pol=None,
 
     query_sia.__doc__ = query_sia.__doc__.replace('_SIA2_PARAMETERS', SIA2_PARAMETERS_DESC)
 
-    def query_tap(self, query, maxrec=None):
+    def query_tap(self, query, maxrec=None, uploads=None):
```

Comment:
Can we make things kwarg only?

```suggestion
    def query_tap(self, query, *, maxrec=None, uploads=None):
```